fission/function.go

```diff
@@ -392,6 +392,11 @@ func fnUpdate(c *cli.Context) error {
 
 	envName := c.String("env")
 	envNamespace := c.String("envNamespace")
+	// if the new env specified is the same as the old one, no need to update package with envname
```

Comment:
Isn't this true for all update params?  E.g. package, build command, etc.?

Reply:
Ok.  Can you put this in a comment?

Reply:
ok, 👍